dbot/dbot/Migrations/20211101202513_InitialCreate.cs

```diff
@@ -0,0 +1,122 @@
+﻿using System;
```

Comment:
Should /Migrations be added to the .gitignore?

Reply:
I think they may need to be kept because the migrations are applied to a db if the program needs to create one automatically. I'm not sure how else the migrations would be created unless the program had access to them.

Reply:
After looking it up it looks like the simplest way to handle migrations is to just leave them in the repo. I am used to a different method which uses a deployment script to apply the migrations, but let's stick with committing them since it's easier.